src/content/docs/browser/new-relic-browser/getting-started/compatibility-requirements-browser-monitoring.mdx

```diff
@@ -227,6 +227,7 @@ Below are known compatibility issues:
 * If your application uses AngularJS and you want to use browser's [SPA monitoring](/docs/browser/single-page-app-monitoring/get-started/welcome-single-page-app-monitoring) capabilities, Zone.js versions 0.6.18-0.6.24 are not compatible with the SPA agent.
 * The html2pdf.js library is not compatible with the SPA agent.
 * Angular versions 12 through 14 contains a limitation around the size of individual script elements contained within the index HTML source. See [this troubleshooting page](/docs/browser/single-page-app-monitoring/troubleshooting/angular-truncated-copy-paste-snippet) for more information.
+* Next.JS has a known incompatibility since version 13.3.2-canary.2 when using the app router that results in links on the page not working. See [this troubleshooting page](/docs/browser/single-page-app-monitoring/troubleshooting/nextjs-app-router-links-broken) for more information.
```

Comment:
```suggestion
* Next.JS has a known incompatibility since version 13.3.2-canary.2 where using the app router can result in links on the page not working. See [this troubleshooting page](/docs/browser/single-page-app-monitoring/troubleshooting/nextjs-app-router-links-broken) for more information.
```